core/primitives/src/runtime/apply_state.rs

```diff
@@ -41,4 +44,6 @@ pub struct ApplyState {
     pub evm_chain_id: u64,
     /// Data collected from making a contract call
     pub profile: crate::profile::ProfileData,
+    #[cfg(feature = "protocol_feature_block_hash_host_fn")]
```

Comment:
Can you add description here?

Reply:
Done in https://github.com/near/nearcore/pull/4273/commits/c095edd5af317348d6d1249c6a193ae4c8f769bf